### DISCUSSION TITLE
Star range slider should be expressed as a directionless range instead of two directional arguments

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20131] [Ideas]
currently, the first dot on the star range slider must always be first, and the second dot must always be second (s1 <= star rating <= s2). This should be changed so you can set one and the other independently and the game would take the range inbetween. Just a small change, but could make setting the star range a little easier.

**Comment by peppy** [comment]
I don't really understand the request here.

**Reply by Theighlin** [reply]
I think he's saying that you should be able to move the 2 dots accross the whole slider without pushing each other, and the game will automatically understand that the zone between them is the desired star range filter.
I don't have a strong opinion on this but i think current behaviour is fine.

**Reply by peppy** [reply]
I intentionally made it "push" the other, and don't foresee that changing (it's a more common / accepted UX pattern).
